proto/def/contract.proto

```diff
@@ -31,8 +36,8 @@ message Egress {
     google.protobuf.Empty replyToOriginalTopic = 4;
   }
 
-  // Dead letter is where the event is sent when something goes wrong
-  string deadLetter = 5;
+  // Egress specific config. Overrides an eventual resource wide egress config
+  EgressConfig egressConfig = 5;
```

Comment:
Can we add this later since we don't it now?

Reply:
+ we don't use it on the data plane, and I would avoid adding more to this already big PR.

Reply:
do you mean the EgressConfig type of the fact that there is one in Egress?

Reply:
> the fact that there is one in Egress

Reply:
The type makes sense to me.